### ISSUE TITLE
Add support for Nanazip so that compressed and comic book files can be read.

### ISSUE THREAD

**Issue by DisasterInbound** [Ruben2776/PicView#134]
**Is your feature request related to a problem? Please describe.**
I use [Nanazip](https://github.com/M2Team/NanaZip) which is a fork of 7zip. I tried opening a zip and cbz file but I got an "unknown error" message

**Describe the solution you'd like**
I'm not a developer but since Nanazip has some part of the same code as 7zip maybe it should be possible to add it here.

**Describe alternatives you've considered**
N/A

**Additional context**
Thanks a lot for making this app. I have been looking for an app that could replace the one I had and this one works so well

**Comment by Ruben2776** [comment]
Hey there!

I've made an update at SourceForge. Would you mind checking if this fixes it for you?

https://sourceforge.net/projects/picview/files/%23134/

**Comment by DisasterInbound** [comment]
Hey, thanks for the reply. Happy new year!

I tried unzipping it into a new folder and also into the old folder where PicView was installed and I got this error message in both tries.

![3HNU56vFcO](https://github.com/Ruben2776/PicView/assets/64549167/3f92dd29-c70b-4921-965b-d066e90fb057)

Did I do this incorrectly?

**Comment by Ruben2776** [comment]
Thanks, happy new year to you as well.

I've had an issue with an installation script that zipped the files incorrectly, so some folders were missing from the build.

Can you try and download the new version 2.2? It has the changes in it as well.

**Comment by DisasterInbound** [comment]
Thanks. I tried v2.2 but what is does is open the Nanazip folder into Nanazip itself.

**Comment by Ruben2776** [comment]
@DisasterInbound 
Support for Nanazip has been added in 2.2.1

**Comment by DisasterInbound** [comment]
Hey @Ruben2776 thanks for the update again and sorry to bring (late) bad news, but I still have an issue when trying to open files
I get the following:
![NanaZipG_VdHlz2aAnJ](https://github.com/Ruben2776/PicView/assets/64549167/b475c4f7-68cb-47aa-a8fa-bcc95a8a6933)

Initially I thought it was because the app was installed to C://Program Files and there was no write permissions but I have uninstalled and unzipped the files outside the Program Files folder but I still get this error.

I tried with different .zip, .rar and .7z files if that matters.


Also while testing this I had a question. Does this support compressed files that are password protected?
